tests/core/filtering/test_contract_data_filters.py

```diff
@@ -45,52 +50,41 @@ def fixed_values(draw):
 
 @st.composite
 def array_values(draw):
-    matching = draw(st.lists(elements=st.binary(min_size=2, max_size=2)))
+    matching = draw(st.lists(elements=st.binary(min_size=2, max_size=2), max_size=35, min_size=1))
```

Comment:
> Hypothesis has found a failing example, when the event array argument has 38 or more elements. Ive set the max number of elements to 35. This is probably a eth-tester issue.

Did you find out any more about this? Does a filter with 38 elements work on, say, `geth --dev`?

I hate to have this constraint on the test, until we know what's going on.